src/commands/apps/info.js

```diff
@@ -18,7 +18,7 @@ function * run (context, heroku) {
       }),
       dynos: heroku.get(`/apps/${app}/dynos`).catch(() => []),
       collaborators: heroku.get(`/apps/${app}/collaborators`).catch(() => []),
-      pipeline: heroku.get(`/apps/${app}/pipeline-couplings`).catch(() => null)
+      pipeline_coupling: heroku.get(`/apps/${app}/pipeline-couplings`).catch(() => null)
```

Comment:
I don't know what the standard is in this repo, but I would've camel cased `pipelineCoupling` here.

Reply:
@appleton I thought about that, but this is *also* to serialize it in the json output. Considered leaving it with the underscore to respect current format like `updated_at`. I could still convert it to underscore when rendering the json?

Reply:
Oh that makes sense. I'd leave it alone in that case.